lang/funcs/operator_polyfunc.go

```diff
@@ -374,11 +375,15 @@ func LookupOperator(operator string, size int) ([]*types.Type, error) {
 		return nil, fmt.Errorf("operator not found")
 	}
 	results := []*types.Type{}
-
```

Comment:
code is perfect, please re-add this nl, and change commit prefix to `lang` instead `language`.